app/src/main/AndroidManifest.xml

```diff
@@ -3,7 +3,11 @@
     package="de.robv.android.xposed.installer"
     xmlns:android="http://schemas.android.com/apk/res/android">
 
-    <uses-permission android:name="android.permission.ACCESS_SUPERUSER"/>
+    <uses-sdk
```

Comment:
As i said im not experienced with git(hub). can't you just take the lines you like?